sql-to-dbsp-compiler/SQL-compiler/src/main/java/org/dbsp/sqlCompiler/compiler/backend/rust/ToRustHandleVisitor.java

```diff
@@ -173,6 +174,36 @@ private void generateRenameMacro(String tableName, DBSPTypeStruct type, boolean
                 .decrease()
                 .append("});")
                 .newline();
+
+        this.builder.append("serialize_table_record!(");
+        this.builder.append(type.sanitizedName)
+                .append("[")
+                .append(type.fields.size())
+                .append("]{")
+                .increase();
+        first = true;
+        for (DBSPTypeStruct.Field field: type.fields.values()) {
+            if (!first)
+                this.builder.append(",").newline();
+            first = false;
+            String name = field.name;
+            boolean quoted = field.nameIsQuoted;
+            if (output) {
+                name = name.toUpperCase(Locale.ENGLISH);
```

Comment:
I never specified the locale explicitly before. Perhaps for consistency we should keep it without an argument, otherwise there may be strange results.

Reply:
I copy-pasted this.  But I think this is better than have non-deterministic behavior based on the server locale (presumably that's the default behavior?)

Reply:
Like I said, I copied this from existing code in the same file. So I'm not sure what consistent would mean here.

Reply:
we should remove Locale.English from both places. Sorry.

Reply:
This might break some of the exotic tests, so I'm not keen on doing it in this PR.  Are there many other places in the compiler where we use `toUpperCase` on column names?

Reply:
This is the only place where we supply a locale.